docker/grafana/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM grafana/grafana:10.1.2
+
+ADD ./infrastructure/grafana/ /etc/grafana/provisioning/
+
+USER root
+
+RUN sed -i 's/http:\/\/prometheus:9090/https:\/\/monitoring-prometheus.internal.cke-cs-dev.com/g' /etc/grafana/provisioning/datasources/prometheus-source.yaml
```

```suggestion
RUN sed -i 's/http:\/\/prometheus:9090/https:\/\/$PROMETHEUS_ADDRESS/g' /etc/grafana/provisioning/datasources/prometheus-source.yaml
```

It would be nice to have a single `Dockerfile`. Is it a lot of work to adjust orb script for ability to passing env variable (address or just flag `local/prod`) to Dockerfile during build process 🤔 ?